azure-pipelines.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+# Imports the shared Azure CI tools
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: shotgunsoftware/tk-ci-tools
+      ref: refs/heads/run_tests
+      endpoint: shotgunsoftware
+
+# Configures CI to be triggered by changes to any branches and on tag creation.
+trigger:
+  branches:
+    include:
+    - '*'
+  tags:
+    include:
+    - v*
+
+# This pulls in a variable group from Azure. Variables can be encrypted or not.
+variables:
+- group: deploy-secrets
+
+jobs:
```

Comment:
I think it might be worthwhile to just add one more comment above the jobs section to be explicit about what's going on there as well.  Maybe something to the effect of "Finally, run the jobs specified by build-pipeline.yml in the tk-ci-tools repo"?